pkg/xds/envoy/listeners/v3/tracing_configurer.go

```diff
@@ -82,6 +89,37 @@ func datadogConfig(cfgStr *structpb.Struct, backendName string, serviceName stri
 	return tracingConfig, nil
 }
 
+func otelConfig(cfgStr *structpb.Struct) (*envoy_trace.Tracing_Http, error) {
+	cfg := mesh_proto.ZipkinTracingBackendConfig{}
+	if err := proto.ToTyped(cfgStr, &cfg); err != nil {
+		return nil, errors.Wrap(err, "could not convert backend")
+	}
+
+	otelConfig := envoy_trace.OpenTelemetryConfig{
+		ServiceName: "unknown_service:TODO(frzifus):envoy",
+		GrpcService: &envoy_corev3.GrpcService{
+			TargetSpecifier: &envoy_corev3.GrpcService_GoogleGrpc_{
+				GoogleGrpc: &envoy_corev3.GrpcService_GoogleGrpc{
+					TargetUri: cfg.Url,
+					// TODO(frzifus): support more options?
+				},
+			},
+		},
+	}
+
+	otelConfigAny, err := proto.MarshalAnyDeterministic(&otelConfig)
+	if err != nil {
+		return nil, err
+	}
+	tracingConfig := &envoy_trace.Tracing_Http{
+		Name: "envoy.opentelemetry",
```

Comment:
Isn't it supposed to be envoy.tracers.opentelemetry as described [here](https://www.envoyproxy.io/docs/envoy/v1.22.0/api-v3/config/trace/v3/http_tracer.proto#config-trace-v3-tracing-http) ?

Reply:
y, seems like :)